app/js/lib/view_handler/ppl_recurrent_warning.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for showing and hiding elements if the update value matches a regular expression
+	 * @class
+	 */
+	PPLRecurrentWarning = {
```

Comment:
It becomes pretty clear what this thing does and under what circumstances when looking at the code. The name might not be self-explanatory, though. Maybe something like `PaymentTypeNotice` or at least `PayPalRecurrentWarning`? Not a big thing, though. You might as well ignore my comment.